apps/web/src/components/Wallet.vue

```diff
@@ -203,6 +204,7 @@ const {
 const { deposit, getPools, withdraw } = useSSV()
 
 onMounted(async () => {
+  console.log('got to Wallet.vue onMounted')
```

Comment:
Can probs delete too.

Reply:
This is your component, do you need that log or not? @ccali11